Vikram's "Airflow 2.0 Scheduler" Blog Post

paolaperaza opened the pull request:
Alrighty, PR here for @vikramkoka's blog post around Airflow 2.0's refactored Scheduler. Original copy here: https://docs.google.com/document/d/1FQ6hj9KP6hhkhS5ifsV3fEqjqtc1PKpbPZqKEBSLHWs/edit#

Currently missing:

- [x] Add Hero
- [x] Add 2 Images
- [x] Author ok?
- [x] Final Review

@petedejoy @sunkickr Can you two read this over to make sure the front-matter, file name etc. is all set from a website standpoint? I'll have myself and @vikramkoka give the actual content a final read otherwise to make sure we're good to go.

vikramkoka reviewed (approved):
Thank you for the quick feedback incorporation! This looks really good!

paolaperaza commented:
@sunkickr @petedejoy For some reason the 2 images I added aren't rendering in my code editor - not sure if they're supposed to if said images haven't been merged yet (they're on this branch).

Totally might have done something wrong on my end - let me know if I did? Copied the same format by which we're referencing other images in the assets repo, although I _did_ add an extra directory on top of it for tidiness sake (i.e. I added a "airflow-2-scheduler" folder in "assets", and put the 2 images in said folder).

sunkickr commented on posts/airflow-2-scheduler.md:
```diff
@@ -0,0 +1,204 @@
+| 2                     | 12                 | 541                        | 492                   | 578                    |
+| 3                     | 12                 | 698                        | 632.5                 | 774                    |
+
+![Airflow Scheduler Task Throughput](..assets/airflow-2-scheduler/scheduler-task-throughput.png)
```
the image url needs to be ../assets/airflow-2-scheduler/scheduler-task-throughput.png instead of ..assets/airflow-2-scheduler/scheduler-task-throughput.png

sunkickr commented on posts/airflow-2-scheduler.md:
```diff
@@ -0,0 +1,204 @@
+
+A sample Airflow Scheduler HA deployment on a set of Virtual Machines is shown below. The Airflow 1.x deployment model is on the left, with the Airflow 2.0 scalable scheduler deployment model on the right.
+
+![Airflow Scheduler Deployment Model](..assets/airflow-2-scheduler/scheduler-deployment-model.png)
```
The image url needs to be changed like the one above ti ../assets

sunkickr commented on posts/airflow-2-scheduler.md:
```diff
@@ -0,0 +1,204 @@
+
+As members of the Airflow community, we are excited for a new frontier!
+
+
```
I don't know how all these extra lines at the bottom will render in Gatsby. Gatsby seems to take markdown more literal than most

paolaperaza replied:
Removed spaces

sunkickr commented on posts/airflow-2-scheduler.md:
```diff
@@ -0,0 +1,204 @@
+
+To sufficiently test this, without skewing numbers based on the actual task “work” time, we have chosen to benchmark using a simple PythonOperator task with a trivial execution time. The benchmarking configuration was: Celery Workers, PostgreSQL DB, 1 Web Server.
+
+Results for _task throughput_ (metric explained above) using Airflow 2.0 beta builds, run with 5,000 DAGs, each with 10 parallel tasks on a single Airflow deployment. The benchmark was performed on Google Cloud and each Scheduler was run on a [n1-standard-1 machine type](https://cloud.google.com/compute/docs/machine-types#n1_machine_types).
```
Gatsby messes up links with underscores in them because it will italicize. This link  needs escape chars like \ in front of _
